met-api/src/met_api/models/submission.py

```diff
@@ -53,6 +53,16 @@ def get_by_survey_id(cls, survey_id) -> List[SubmissionSchema]:
     @classmethod
     def create(cls, submission: SubmissionSchema, session=None) -> Submission:
         """Save submission."""
+        has_comments = cls.__check_if_submission_has_comments(submission)
```

Comment:
can you add some comments why we are setting to approved if no comments?
just to explain the logic..

Reply:
added the comments, thanks